src/components/safe-apps/SafeAppIcon/index.tsx

```diff
@@ -1,17 +1,19 @@
 import { type ReactElement, memo } from 'react'
 
-const getIframeContent = (src: string): string => {
-  return `<style>
-    html, body {
-      padding: 0;
-      margin: 0;
-      background-color: transparent;
-      height: 100%;
-    }
-    body {
-      background: url('${src}') center center/contain no-repeat;
-    }
-  </style>`
+const APP_LOGO_FALLBACK_IMAGE = `/images/apps/app-placeholder.svg`
+
+const getIframeContent = (url: string, alt: string, width: number, height: number): string => {
+  return `
+     <body style="margin: 0;">
+       <img src=${url} alt=${alt} width="${width}" height="${height}" />
```

Comment:
I would wrap `url` and `alt` in double-quotes, and escape any double quotes in those strings. Otherwise, we have a classic XSS hole here.

Reply:
Actually, I would hardcode smth in the alt tag like "app logo". The name of the app is already on the card anyway.

Reply:
Good spot regarding xss. I've checked other platforms with lists to see how they implement alt attributes and seems like they don't lol (checked chrome web store, slack store)